File: Placements.DataAccess/placement/Models/Collegejobpostingschedule.cs
using System;
using System.Collections.Generic;

namespace Placements.DataAccess.Placement.Models;

public partial class Collegejobpostingschedule
{
    public long Id { get; set; }

    public long? CollegeId { get; set; }

    public long? JobPostingId { get; set; }

    public DateTime? ScheduledDate { get; set; }

    public virtual Campusregistration? College { get; set; }

    public virtual Jobposting? JobPosting { get; set; }
}

File: web-app/src/app/services/types/Collegejobpostingschedule.ts
import { Campusregistration } from "./Campusregistration";
import { Jobposting } from "./Jobposting";

export type Collegejobpostingschedule = {
    Id: number;
    CollegeId?: number;
    JobPostingId?: number;
    ScheduledDate?: Date;
    College: Campusregistration[];
    JobPosting: Jobposting[];
}